boards/sim/sim/sim/configs/btuart/defconfig

```diff
@@ -1,73 +0,0 @@
-#
```

Comment:
Please provide a config which excercises the new interface instead of just removing the previous one.

Reply:
https://github.com/apache/incubator-nuttx/pull/3754/commits/5ef77384fd30936070f45bfb710861aa3089db96